manifests/cf-manifest/deployments/040-cf-properties.yml

```diff
@@ -448,85 +436,96 @@ properties:
       watcher:
         debug_addr: 0.0.0.0:17015
 
-base_job_templates:
-  brain:
-    - name: consul_agent
-      release: cf
-    - name: auctioneer
-      release: diego
-    - name: converger
-      release: diego
-    - name: metron_agent
-      release: cf
-  cell:
-    - name: consul_agent
-      release: cf
-    - name: rep
-      release: diego
-    - name: garden
-      release: garden-linux
-    - name: rootfses
-      release: diego
-    - name: metron_agent
-      release: cf
-  cc_bridge:
-    - name: consul_agent
-      release: cf
-    - name: stager
-      release: diego
-    - name: nsync
-      release: diego
-    - name: tps
-      release: diego
-    - name: cc_uploader
-      release: diego
-    - name: metron_agent
-      release: cf
-  route_emitter:
-    - name: consul_agent
-      release: cf
-    - name: route_emitter
-      release: diego
-    - name: metron_agent
-      release: cf
-  access:
-    - name: consul_agent
-      release: cf
-    - name: ssh_proxy
-      release: diego
-    - name: metron_agent
-      release: cf
-    - name: file_server
-      release: diego
-  database:
-    - name: consul_agent
-      release: cf
-    - name: bbs
-      release: diego
-    - name: metron_agent
-      release: cf
-  colocated:
-    - name: consul_agent
-      release: cf
-    - name: auctioneer
-      release: diego
-    - name: bbs
-      release: diego
-    - name: cc_uploader
-      release: diego
-    - name: converger
-      release: diego
-    - name: file_server
-      release: diego
-    - name: metron_agent
-      release: cf
-    - name: nsync
-      release: diego
-    - name: route_emitter
-      release: diego
-    - name: ssh_proxy
-      release: diego
-    - name: stager
-      release: diego
-    - name: tps
-      release: diego
+meta:
+  environment: ~
+  default_quota_definitions:
+    default:
+       memory_limit: 2048
+       total_services: 10
+       non_basic_services_allowed: false
+       total_routes: 1000
+  secrets:
+    consul_ca_cert: (( grab secrets.bosh_ca_cert ))
+    bbs_ca_cert: (( grab secrets.bosh_ca_cert ))
+  base_job_templates:
```

Comment:
I think this should be renamed to `diego_templates` to make it clear what it's scope is.